src/components/Details/Details.tsx

```diff
@@ -23,23 +24,22 @@ interface SelectedMarketProps {
   id: string;
 }
 
-export const Details: React.FC<SelectedMarketProps> = ({
-  selectedMarket,
-  id
+export const Details: React.FC<SelectedMarketProps> = ({ 
+  selectedMarket, 
+  id 
 }) => {
-  const openSeason = `Season: ${selectedMarket?.schedule[0].season}`;
-
-  const daysAndTimes = selectedMarket?.schedule.map((sched, index) => {
```

Comment:
i think we are going to need this schedule.map due to some markets having multiple daysOfWeek in an array